inc/Server.hpp

```diff
@@ -35,6 +34,7 @@ class Server {
 	int m_serverSock;
 	int m_epfd;
 	std::map<int, std::string> m_requestStrings;
+	std::map<int, time_t> m_connectionLastActive;
```

Comment:
Since there are several datapoints referring to a specific connection (connected server, time, bytesReceived) I would manage it in its own class like done in #22
I didn't move the buffer String map (yet in #22 ), but would also opt to do so

Reply:
WDYM move the buffer string map?

Reply:
Move the buffer in the Connection class. Like a Connection class has a member buffer

Reply:
Yeah makes sense.